contentcuration/contentcuration/static/js/bundle_modules/base.js

```diff
@@ -1,62 +1,66 @@
-var attachfastclick = require("fastclick");
+var attachfastclick = require('fastclick');
 
-var $ = require("jquery");
-var get_cookie = require("utils/get_cookie");
+var $ = require('jquery');
+var get_cookie = require('utils/get_cookie');
 global.$ = $;
```

Comment:
Seemed a little redundant to use `global` here, since `base.js` is already at the global scope. `$` and `jQuery` are also both defined by just doing `require('jquery')`. I suppose it's best to be explicit though.